o3de/o3de
# feat: add visiblity toggle for MeshComponent (GHI-17832)

## What does this PR do?

add visibility toggle meshComponent

## How was this PR tested?

- verify toggling visibility will hide mesh. 
- reload map and see visibility persists correctly

ref: https://github.com/o3de/o3de/issues/17832
